plugins/synaptics-rmi/fu-synaptics-rmi-device.c

```diff
@@ -772,6 +773,7 @@ fu_synaptics_rmi_device_disable_irqs (FuSynapticsRmiDevice *self, GError **error
 	if (!fu_synaptics_rmi_device_write (self,
 					    priv->f01->control_base + 1,
 					    interrupt_disable_req,
+						FALSE,
```

Comment:
all the indenting is wrong -- if you can fix that I can fix up the failure for you.